src/Altinn.App.Core/Features/Payment/Processors/Nets/NetsMapper.cs

```diff
@@ -62,6 +62,54 @@ internal static class NetsMapper
         };
     }
 
+    /// <summary>
+    /// Map from out Payer type to NetsCheckoutConsumerDetails.
+    /// </summary>
+    public static NetsCheckoutConsumerDetails? MapConsumerDetails(Payer? payer)
+    {
+        if (payer == null)
+        {
+            return null;
+        }
+
+        if (payer.PrivatePerson != null && payer.Company != null)
```

Comment:
I saw this after my initial comment. This could be payer.Validate(), but the current approach is fine as well